backend/graphql/types/courseType.ts

```diff
@@ -16,6 +18,8 @@ const courseType = gql`
     image: String
     previewImage: String
     lessons: [ID!]!
+    private: Boolean!
+    published: Boolean!
```

Comment:
I thought we decided that these shouldn't be mandatory - both defaulting to false? This is the request DTO, which is going to be sent from the frontend to the backend.